task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,12 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    if (num % 3 == 0) {
+        if (num % 5 == 0) return 'FizzBuzz'
+        return 'Fizz'
+    }
+    if (num % 5 == 0) return 'Buzz';
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"